packages/openfeature-web-provider/src/ConfidenceWebProvider.test.ts

```diff
@@ -267,67 +296,34 @@ describe('ConfidenceProvider', () => {
       });
     });
 
-    it('should return default if the provider is not ready', () => {
-      const actual = instanceUnderTest.resolveBooleanEvaluation(
-        'testFlag.bool',
-        false,
-        dummyEvaluationContext,
-        dummyConsole,
-      );
-
-      expect(actual).toEqual({
-        value: false,
-        errorCode: ErrorCode.PROVIDER_NOT_READY,
-        reason: 'ERROR',
-      });
+    it('should throw if the provider is not ready', () => {
+      expect(() =>
+        instanceUnderTest.resolveBooleanEvaluation('testFlag.bool', false, dummyEvaluationContext, dummyConsole),
+      ).toThrow(new GeneralError('Provider not ready'));
```

Comment:
Seems like `PROVIDER_NOT_READY` [error](https://openfeature.dev/specification/types#error-code) has not been added [to the SDK](https://github.com/open-feature/js-sdk/tree/main/packages/shared/src/errors) yet. Perhaps we can open a PR in the SDK repo and then throw the related exception here?

Reply:
Yes, i have that change ready to go actually! :)